packages/teleport/src/Nodes/NodeAdd/ByScript.tsx

```diff
@@ -43,7 +47,7 @@ export default function ByScript(props: Props) {
         <br />
         Script will be valid for{' '}
         <Text bold as={'span'}>
-          {expires}.
+          {expiresText}.
```

Comment:
Lets change the entire sentence to:

"Use below script to add a server to your cluster. The script will be valid for 1 hour"

Reply:
revised 
![Screenshot from 2020-10-19 10-05-42](https://user-images.githubusercontent.com/43280172/96488338-ad2b8380-11f2-11eb-83f7-bbcacdfc46d1.png)